test/test_tamper_evident.py

```diff
@@ -73,7 +73,7 @@ def test_editing_is_detected(self):
         self.assertFalse(tef.validate())
 
     def test_oneline_file_is_detected(self):
-        filename = self.write_tamper_evident(b"Am I OK?")
+        filename = self.write_tamper_evident("Am I OK?")
```

Comment:
All of the changes in this file are basically saying to use unicode strings in python 3, and bytestrings in python 2. That seems questionable to me. `write_tamper_evident` should expect one or the other.

Notably, the `TamperEvidentFile.write` method says that the string should be unicode, which it wasn't before (it was bytes before). That seems like it was a bug. I think it would be good to use `u` prefixed strings explicitly here, because that will force unicode strings in both 2 and 3.